content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -772,6 +772,12 @@ The Account Resource supports proprietary systems for:
 
 * Account.identifier.system is [Account Number](#account-number) for financial accounts.
 
+<h4>ChargeItem</h4>
```

Comment:
Do we also need to add ChargeItem up in the proprietary codes section, or is the system all we need?

Reply:
we are going to add that when we implement bill code schedule extension.